packages/skills/carpark_client/handlers.py

```diff
@@ -313,7 +311,7 @@ def handle(self, message: Message) -> None:
         """
         tx_msg_response = cast(TransactionMessage, message)
         if tx_msg_response is not None and \
-                TransactionMessage.Performative(tx_msg_response.get("performative")) == TransactionMessage.Performative.ACCEPT:
+                TransactionMessage.Performative(tx_msg_response.performative) == TransactionMessage.Performative.ACCEPT:
```

Comment:
I think `tx_msg_response.performative` is of type `TransactionMessage.Performative` so no need to convert..